Sources/SymbolKit/SymbolGraph/Symbol/Symbol.swift

```diff
@@ -54,6 +54,9 @@ extension SymbolGraph {
         /// The in-source documentation comment attached to a symbol.
         public var docComment: LineList?
 
+        /// If true, the symbol was created implicitly and not from source.
+        public var isVirtual: Bool
```

Comment:
Is there a benefit to declaring individual symbols as snippets, rather than relying on the module being virtual? Can a virtual module contain non-virtual symbols?

Reply:
Yes, the benefit is that there are cases where you may want to ask if a module is virtual, and other cases where you may want to ask if a symbol is virtual. Having this recorded in both places allows the question to be answered without consulting the graph's relationships, which can be expensive.